dotcom-rendering/src/web/components/Metrics.importable.tsx

```diff
@@ -0,0 +1,115 @@
+import type { ABTest, ABTestAPI } from '@guardian/ab-core';
+import {
+	bypassCommercialMetricsSampling,
+	initCommercialMetrics,
+} from '@guardian/commercial-core';
+import {
+	bypassCoreWebVitalsSampling,
+	initCoreWebVitals,
+} from '@guardian/core-web-vitals';
+import { getCookie } from '@guardian/libs';
+import { dcrJavascriptBundle } from '../../../scripts/webpack/bundles';
+import type { ServerSideTestNames } from '../../types/config';
+import { integrateIma } from '../experiments/tests/integrate-ima';
+import { useAB } from '../lib/useAB';
+import { useAdBlockInUse } from '../lib/useAdBlockInUse';
+import { useOnce } from '../lib/useOnce';
+
+type Props = {
+	enabled: boolean;
+};
+
+const sampling = 1 / 100;
+/** defining this here allows to share this with other metrics */
+const willRecordCoreWebVitals = Math.random() < sampling;
```

Comment:
The sign is corrected here, as it was doing the comparison the wrong way around in #6999